components/popover/popover.vue

```diff
@@ -698,6 +698,11 @@ export default {
     **/
     preventScrolling (e) {
       if (!this.modal) return;
+      const isEventComingWithinPopover = Object.keys(this.$refs).some(refName => {
```

Comment:
Do we need to check all refs here? can't we just check within ref="content"?

Reply:
I think it might work with content only, let me test it.